\documentclass[onecolumn, 11pt,openany]{memoir}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% packages
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\label{packages}
\usepackage{blindtext, color}
\usepackage[usenames,dvipsnames,svgnames,table,xcdraw]{xcolor}
\usepackage{graphicx}  
\usepackage{import}  
\usepackage[english]{babel}
\usepackage[outline]{contour} 
\usepackage{ctable}
\newcolumntype{d}[1]{D{.}{.}{#1}}
\usepackage[utf8]{inputenc}
\usepackage[compact]{titlesec}
\usepackage{pdfpages}
\usepackage{float}
\usepackage{microtype}
\definecolor{lightgray}{gray}{0.9}
\usepackage{multicol}
\usepackage{authblk}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% font and lay-out
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%current font: BT Charter. Find other fonts at http://www.tug.dk/FontCatalogue/
\usepackage[bitstream-charter]{mathdesign}
\usepackage[T1]{fontenc}

%caption layout
\usepackage[format=plain]{caption}
\DeclareCaptionLabelSeparator{bar}{ | }
\captionsetup[figure]{labelfont={bf},name={Figure},labelsep=bar,font=footnotesize}
\captionsetup[table]{labelfont={bf},name={Table},labelsep=bar,font=footnotesize}

%symbols
\usepackage{textcomp}
\usepackage{gensymb}
\usepackage[euler]{textgreek}

% paragraph indent & spacing
\setlength\parskip{\baselineskip}
\setlength\parindent{0pt} % use 0 pt for no indent

% page sizes and margins
\usepackage[a4paper, margin=1in]{geometry}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% hyphenation
% list all hy-phe-na-ted words as they should be hyphenated
% to repress hyphenation place word (or two) in an mbox operator; e.g.:  \mbox{P. infestans} 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\hyphenation{oo-my-ce-tes Phy-toph-tho-ra path-o-gens spo-ran-gium ge-no-me Govers Hoogen Meijer Seidl Kruif Ketelaar Kots Verbeek Judelson Tyler infestans sojae}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% START document
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\label{START}
\begin{document}
\thispagestyle{empty}                   
\frontmatter

\title{Attempts to apply CRISPR/Cas9 for genome editing in \textit{Phytophthora infestans}}
\author[1,2]{Johan van den Hoogen}
\author[1]{Francine Govers}
\begin{scriptsize}
\affil[1]{Laboratory of Phytopathology, Wageningen University, Wageningen, The Netherlands}
\affil[2]{Global Ecosystem Ecology, Institute of Integrative Biology, ETH Zürich, Zürich, Switzerland}
\end{scriptsize}
\date{\vspace{-5ex}}
\maketitle

\subsection{Correspondence}
johan.vandenhoogen@usys.ethz.ch\\
francine.govers@wur.nl

\pagenumbering{arabic}
\pagestyle{plain}

\section{Abstract}
Few techniques have revolutionized the molecular biology field as much as genome editing using CRISPR/Cas9. Recently, a CRISPR/Cas9 system has been developed for the oomycete \textit{Phytophthora sojae}, and since then it has been employed in two other \textit{Phytophthora} spp. Here, we report our progress on efforts to establish the system in the potato late blight pathogen \textit{Phytophthora infestans}. Using the original constructs as developed for \textit{P. sojae}, we did not obtain any transformants displaying a mutagenized target gene. We made several modifications to the CRISPR/Cas9 system to pinpoint the reason for failure and also explored the delivery of pre-assembled ribonucleoprotein complexes. With this report we summarize an extensive experimental effort pursuing the application of a CRISPR/Cas9 system for targeted mutagenesis in \textit{P. infestans} and we conclude with suggestions for future directions.

\begin{multicols}{2}
\section{Introduction}
The oomycete \textit{Phytophthora infestans}, the causal agent of potato and tomato late blight, is an economically important plant pathogen that is difficult to control (Kamoun et al. 2015). Current efforts to identify and functionally analyze genes important for virulence, are hampered by a limited molecular toolbox. Gene knock-outs (KOs) or gene deletions are not possible via homologous recombination, as transgenes are integrated randomly via non-homologous end-joining (NHEJ) (Judelson 1997; Fang \& Tyler 2016). Moreover, \textit{P. infestans }is diploid and heterothallic. Generating sexual progeny is quite challenging and as a result studies based on genetic analyses are scarce (Govers \& Gijzen 2006; Fry 2008). Consequently, to date, functional gene studies in oomycetes primarily rely on gene silencing and overexpression. DNA transformation of the target gene in \textit{P. infestans }can lead to homology dependent gene silencing or overexpression (van West et al. 1999). By fusing a fluorescent tag to the target gene, the overexpression transformants can also be exploited for investigating the subcellular localization of the encoded protein. However, due to the random integration of transgenes and varying levels of silencing and overexpression efficiency, phenotypes often vary between transformed lines, experiments, and labs (Fang \& Tyler 2016). 

Recently, a CRISPR/Cas9 genome editing system has been developed for \textit{Phytophthora sojae }(Fang \& Tyler 2016). This provides an important addition to the molecular toolbox for oomycetes, and makes it up to par with other research areas. For genome editing based on CRISPR/Cas systems, a nuclease, e.g. Cas9, is targeted to the desired DNA sequence by a so-called synthetic guide RNA (gRNA). The 20-nucleotide protospacer directs Cas9 to a specific target DNA site, which must be immediately 5' of a protospacer adjacent motif (PAM) sequence (Ran et al. 2013). When Cas9 is directed to the target site a double-stranded break (DSB) is induced, which can be repaired either by the NHEJ repair mechanism, or via homology-directed repair (HDR) in case a repair template is available (Ran et al. 2013). Because of its low fidelity, NHEJ often leads to insertions or deletions (indels) in the repaired target gene, which can lead to frame shift mutations. HDR, on the other hand, can be exploited to introduce specific modifications such as base substitutions, insertions, or gene deletions. 

So far, the \textit{P. sojae} CRISPR/Cas9 system has been successfully used to mutagenize several genes in \textit{P. sojae} (Fang \& Tyler 2016; Ma et al. 2017), \textit{Phytophthora capsici }(B.M. Tyler, personal communication), and \textit{Phytophthora palmivora} (Gumtow et al. 2017). However, as of to date, no one has reported successful implementation of the system in \textit{P. infestans}. Here, we present a case study for the effectuation of CRISPR/Cas9 for targeted genome editing in \textit{P. infestans}. We chose to target three genes in \textit{P. infestans} for a proof-of-principle study of CRISPR/Cas9-based genome editing in this organism. 

The first target, \textit{Avr1}, encodes an RXLR effector protein recognized by the cognate receptor protein R1 in potato. Recognition of AVR1 by R1 provides resistance to potato, while\textit{ P. infestans} strains lacking \textit{Avr1 }are virulent on potato cultivars harboring resistance gene \textit{R1}. Moreover, R1 fails to recognize several mutated forms of AVR1 (Du et al. 2018). Hence, \textit{P. infestans Avr1} KO lines are expected to gain virulence on \textit{R1} potato, providing a clear phenotype compared to non-edited lines. The second target, \textit{PiTubA2}, is one of the five α-tubulin encoding genes in \mbox{\textit{P. infestans }}(unpublished data). Our aim was to modify the endogenous \textit{PiTub2A }gene\textit{ }in such a way that it encodes a fusion protein that has a GFP tag fused to the N-terminus of PiTUBA2. We therefore designed a HDR construct for gene knock-in at the endogenous \textit{PiTubA2 }locus. A PiTubA2-GFP knock-in line would be ideal for live cell imaging of the microtubule cytoskeleton in \textit{P. infestans}, as it lacks possible non-desirable side effects associated with overexpression and/or ectopic integration of the transgene. The third target, \textit{PiAP5}, encodes an aspartic protease (AP) with an C-terminal GPCR domain (Kay et al. 2011), a protein which is unique for oomycetes (Van den Hoogen et al. 2018). To study to what extent the GPCR domain is important for functioning of the AP domain, we aimed at targeting Cas9 to the central region of the gene that separates the parts encoding the AP domain and the GPCR domain. An indel in this region should result in a truncated protein, which contains the N-terminal AP domain but lacks the C-terminal GPCR domain.

In the initial setup of our experiments, we used the CRISPR/Cas9 system as developed by Fang and Tyler (2016). In this system, the expression of Cas9 is driven by the strong constitutive \textit{Bremia lactucae }Ham34 promoter and the expression of the gRNA\textit{ }by the \textit{P. sojae} RPL41 promoter. The RPL41 promoter was chosen because the U6 small nuclear RNA promoters typically used in CRISPR/Cas9 systems, did not yield any detectable transcript (Fang et al. 2017). To ensure correct release of the gRNA from the transcript, the gRNA sequence was flanked with a hammerhead and a hepatitis delta virus (HDV) ribozyme at the 5' and 3' side, respectively (Fang et al. 2017).

This report summarizes an extensive experimental effort pursuing the application of a CRISPR/Cas9 system for targeted mutagenesis in \textit{P. infestans} and concludes with suggestions for future directions.

\section{Results and discussion}
\subsection{Experimental setup: design of gRNAs}
The first, and perhaps most critical step of setting up a CRISPR/Cas9 experiment, is the design of the gRNAs. The genes of interest were screened for target sites using NGG as the PAM. All gRNAs were scored for on-target activity according to Doench \textit{et al}. (Doench et al. 2014) and for off-targets interactions according to Hsu \textit{et al}. (Hsu et al. 2013). The top 10 best scoring protospacers were manually curated using BLAST analysis on the \textit{P. infestans }reference genome and predictions of the secondary RNA structure of the corresponding gRNAs. For \textit{Avr1}, we selected the three best scoring gRNAs (\textbf{Figure ~\ref{fig1_targets}a}), i.e. no predicted off-target interactions and no strong secondary RNA structure. For \textit{PiTubA2 }and \textit{PiAP5} we selected the highest scoring gRNA (\textbf{Figure ~\ref{fig1_targets}b, ~\ref{fig1_targets}c}).

\begin{figure*}[ht]
\centering
\includegraphics[width=1\textwidth]{Figures/Fig1}
\caption{\textbf{CRISPR loci and HDR constructs.} \textbf{(a)} \textit{PiAvr1}, \textbf{(b) }\textit{PiTubA1}, and \textbf{(c)} \textit{PiAP5}. Orange arrowheads indicate expected DSB sites upon Cas9 nuclease activity; black arrows indicate start codons (ATG); grey blocks mark homologous regions between the genes and the HDR constructs (referred to as repair template or ssODN); interpuncts (•) represent cropped sequences; PAM: Protospacer Adjacent Motif.}
\label{fig1_targets}
\end{figure*}

\subsection{HDR constructs}
To employ the HDR repair pathway for targeted mutagenesis, we used a similar approach as previously used in \textit{P. sojae}. This implies cotransformations of three plasmids, i.e., two plasmids with sequences encoding Cas9 and gRNA, respectively, and the third one containing the HDR repair template (Fang \& Tyler 2016).

For \textit{Avr1} we designed a construct to replace the coding region with \textit{HygB, }a hygromycin-B resistance gene (\textbf{Figure ~\ref{fig1_targets}a}). The inserts were flanked by a 1 kb right flanking arm and a 875 bp left flanking arm that are complementary to the target site for recombination. The reason for the shorter left flanking arm is a \~ 380 bp region in the genome assembly, that is not accessible for sequencing. We opted to use the largest possible flanking region, i.e. 875 bp (\textbf{Figure ~\ref{fig1_targets}a}). For \textit{PiTubA2} we designed a construct to knock-in GFP just before the start codon of the open reading frame (\textbf{Figure ~\ref{fig1_targets}b}). The repair template was designed as such that the PAM would be disrupted upon HDR repair. 

For a number of organisms, higher efficiency of HDR repair has been reported when making use of single-stranded oligodeoxynucleotides (ssODN) as repair template, instead of plasmid DNA (Chen et al. 2011). We designed a ssODN repair template for \textit{AP5}, comprising an 11 bp insert flanked by two 75 nt homology arms. The insert is designed in such a way that there is a stop codon (TAG) in each of the three open reading frames (\textbf{Figure ~\ref{fig1_targets}c}). Further, the ssODN and gRNA were designed as such, that introduction of the 11 bp construct would disrupt the protospacer and consequently withhold Cas9 from further processing the target site. The expected protein product from the truncated gene contains the AP domain but lacks the GPCR domain.

\subsection{Testing the detection limit }
We next examined the sensitivity of our screening methods. To do so, we simulated the situation that an unknown fraction of transformants contains a Cas9-induced mutation. This is expected to be the case in a sample from pooled transformants after transformation; screening these for mutations can give an idea about the efficiency of targeted mutagenesis. First, we constructed an \textit{Avr1} amplicon (ΔAvr1) containing the 29 bp deletion that is expected after Cas9 cleavage at sites gRNA2 and gRNA3 (\textbf{Figure ~\ref{figS1_detect}a}). Next, varying molar ratios of \textit{Avr1} and ΔAvr1 amplicons were annealed and incubated with T7 endonuclease I (T7EI), which is an enzyme that specifically digests mispaired DNA. We found that the detection limit is just over a molar ratio of 95:5 (i.e. 5\% mutated amplicons) (\textbf{Figure ~\ref{figS1_detect}b}).

In parallel, PCR amplicons were sequenced. When sequencing an amplicon that is obtained from a PCR on a pool of transformants of which a certain fraction underwent CRISPR-induced mutagenesis, the resulting sequence chromatogram will contain a certain amount of ‘noise’, or background signals. These aberrant chromatogram peaks reflect the approximate frequency of amplicons with an indel, which can be estimated with an analysis tool such as TIDE (Brinkman et al. 2014). TIDE quantifies the editing efficacy by calculating the statistical probability of finding background signals in the chromatogram of a sample compared to finding them in a reference sequence chromatogram. A range of molar ratios of \textit{Avr1}:ΔAvr1 was sequenced, and by using \mbox{T30-2} \textit{Avr1} as a reference, we analyzed the sequence chromatograms for mismatches. We found the lowest statistically significant detection at a molar ratio of 998:2 (i.e. 0.2\% of the sequences having the deletion) (\textbf{Figure ~\ref{figS1_detect}c}). 

These detection limits are well below the observed frequencies of CRISPR-induced mutations in \textit{P. sojae}, where up to 80\% of the analyzed transformants have undergone NHEJ (Y.F. Fang, personal communication). Assuming that the CRISPR/Cas9 system is functional in \textit{P. infestans}, our detection assays are sufficiently sensitive to reveal induced mutations. It should be noted however, that in the simulations described above only a single mutation (i.e. a 29 bp deletion) was present. In an actual sample of pooled transformed \textit{P. infestans} protoplasts, a mix of differently sized indels is expected and the detection limit will be higher. Nonetheless, we assume that our detection assays are sufficiently sensitive to detect CRISPR-induced mutations in pooled \textit{P. infestans }transformed protoplasts.


\subsection{\textit{In vitro} activity assay}
To examine the functionality of the designed gRNAs, we performed an \textit{in vitro }cleavage assay of target DNA. This assay can give an indication of the \textit{in vivo} efficacy of the gRNA for Cas9 activity. We used purified Cas9 protein and \textit{in vitro} transcribed gRNA in equimolar amounts, in combination with a PCR product as target DNA. We observed Cas9 activity on \textit{Avr1} using gRNA1 and gRNA3 (\textbf{Figure ~\ref{fig2_cleaving}a}). In contrast, gRNA2 failed to direct Cas9 to the target site (\textbf{Figure ~\ref{fig2_cleaving}a}). Examination of the sequence revealed a mistake in the design of the initial construct. When corrected gRNA2 also showed activity (not shown). We did not observe Cas9 activity on \textit{PiTubA2 }using gRNA8, nor on \textit{PiAP5} using gRNA183 (\textbf{Figure ~\ref{fig2_cleaving}b}). In contrast to the initial gRNA2 on \textit{Avr1}, we could not detect mistakes in the design of these two gRNAs. Consequently, judging from their inability to guide Cas9 \textit{in vitro}, it is likely that also the \textit{in vivo} efficacies of gRNA8 and gRNA183 are limited. Other gRNAs for these target genes may have an increased activity, but due to time constraints we have not been able to test alternative gRNAs for \textit{PiTubA2 }and \textit{PiAP5}. 

\begin{figure*}[hb]
\centering
\includegraphics[width=0.78\textwidth]{Figures/Fig2}
\caption{\textbf{In vivo cleavage assay. }\textbf{a)} gRNAs 1, 2, and 3 with Avr1 as target DNA. \textbf{b)} gRNA8 with PiTubA2 as target DNA (\textbf{right}) and gRNA183 with PiAP5 as target DNA (\textbf{left}).}
\label{fig2_cleaving}
\end{figure*}

\subsection{\textit{In vivo} CRISPR/Cas9 in \textit{P. infestans}}
Next, we set out to test the \textit{in vivo }editing efficacy of the system. To this end, we first cloned the respective gRNAs into the expression plasmid pYF2.3-gRNA (Fang \& Tyler 2016). We performed several transformations of \textit{P. infestans} with plasmids pYF2.2-Cas9 and pYF2.3-gRNA and obtained numerous transformants resistant to the selection antibiotic geneticin. In these transformants, the presence of indels as a result of the presumed Cas9 activity was monitored by T7EI digestion and sequencing analysis. However, contrary to expectations set by the positive results of the use of CRISPR/Cas9 in \textit{P. sojae}, \textit{P. capsici}, and \textit{P. palmivora, }the analyses did not reveal any lines in which the target gene was mutagenized. In some cases, the sequence chromatogram showed ambiguities near the target site, raising the hope that the mutagenesis was successful. More detailed analysis however, revealed that these lines were identical to the recipient isolate and should be considered as false-positives. Further trials focused at targeting \textit{Avr1 }included transformations combining two or three gRNA-expressing vectors along with Cas9, but in all cases the results were negative. 

In order to induce specific mutations in \textit{P. infestans}, we performed experiments where plasmid DNA containing a HDR repair template for \textit{Avr1 }was co-transformed with the CRISPR/Cas9 constructs. Even though several hygromycin-B resistant colonies were obtained, and the presence of \textit{HygB }could be detected by PCR on genomic DNA, we were not able to detect integration of the transgene at the endogenous genomic locus. Possibly, the hygromycin-B resistance gene was integrated ectopically, or resistance was conferred by transcription from non-integrated plasmid DNA. Similarly, we did not obtain transformed lines with GFP-encoding sequences inserted at the 5' end of \textit{PitubA2}, or \textit{PiAP5}-mutants with the stop codon insertion.

For the target genes \textit{PiTubA1} and \textit{PiAP5 }only a single gRNA was used. This and the fact the both gRNAs did not prove to be effective in the \textit{in vitro} activity assay, are most likely the reasons for the absence of observable Cas9 activity. However, also in the case of \textit{Avr1 }as target gene, where all three gRNAs proved functional \textit{in vitro}, Cas9 activity appeared to be absent.

An obvious explanation for failure is poor expression of Cas9. To assess the expression level of Cas9, reverse-transcription PCR was performed on RNA isolated from mycelium of transformed lines. In several lines expression of Cas9 could be detected (\textbf{Figure ~\ref{figS2_RT}}). Consequently, it is unlikely that lack of expression of Cas9 is the prime reason for failure of the system. As yet, we have not monitored the presence of the gRNA transcripts. Altogether, these findings led us to conclude that the CRISPR/Cas9 system in \textit{P. infestans }is not functioning, at least not up to the expected efficiency.

\subsection{Modifications to the CRISPR/Cas9 system}
It is hard to pinpoint the main cause for failure of the system, as multiple factors may have a role. Hence, we set out to make modifications at different steps in the process. Below we describe several modifications and alternative approaches in an effort to effectuate the CRISPR/Cas9 system in \textit{P. infestans}.  

\subsubsection{Alternative vectors}
In the original \textit{P. sojae }CRISPR/Cas9 system, there is no selectable resistance marker present on the plasmid encoding the gRNA. However, in practice, cotransformations of plasmids usually result in integration of both plasmids (R. Weide, personal communication). Consequently, geneticin-resistant colonies are expected to also have incorporated the gRNA plasmid. Nevertheless, we decided to test whether we would be able to observe CRISPR/Cas9 activity by using a gRNA-encoding plasmid containing \textit{HygB }as a resistance marker. To do so, we introduced the \textit{HygB} in pYF2.3-gRNA-Ribo, replacing \textit{eGFP} from the vector and yielding pJH2.4-gRNA-Ribo-Hyg (\textbf{Figure ~\ref{figS3_Plasmids}}). In this vector backbone, the respective gRNAs for the three target genes were cloned. Along with the Cas9 encoding plasmid, the resulting plasmids were used for cotransformations of \textit{P. infestans}. Even though we obtained numerous hygromycin-B-resistant tranformants, in none of the screened transformants we could detect mutagenized target genes.

In another attempt, we cloned the gRNA expression components in pYF2.2-Cas9, resulting in the 'all-in-one' vector pJH2.5-Cas9-gRNA (\textbf{Figure ~\ref{figS3_Plasmids}}). Numerous geneticin-resistant transformants were obtained and PCR analysis confirmed the presence of the Cas9 and gRNA genes. Unfortunately, no mutagenized target genes were observed. Further, we constructed a vector containing both gRNA2 and gRNA3 (both targeting \textit{Avr1}), but also here we did not obtain transformed lines with the expected deletion between the two target sites. 

\subsubsection{Nuclear localization sequence}
Nuclear localization of Cas9 is essential. Oomycetes contain distinct nuclear localization sequence (NLS) signals, and commonly used mammalian NLS signals are not efficient in \textit{P. sojae }(Fang \& Tyler 2016; Fang et al. 2017). To overcome this, Fang \& Tyler (2016) fused a synthetic NLS derived from a \textit{P. sojae} bZIP transcription factor (TF) to Cas9. This NLS, further referred to as PsNLS, showed strong nuclear localization in \textit{P. sojae} (Fang et al. 2017), and it is expected that it will perform likewise in \textit{P. infestans}. To test the nuclear localization of Cas9, we obtained several transformed \textit{P. infestans} lines carrying a PsNLS-Cas9-GFP fusion construct but unfortunately, for unknown reasons none of these lines showed fluorescence. Hence, we were unable to determine the localization of Cas9 in \textit{P. infestans}. Anticipating that PsNLS is not able to target Cas9 to the nucleus in \textit{P. infestans}, we set out to test a different NLS. Using a BLAST search, we identified the \textit{P. infestans} homolog of the \textit{P. sojae }bZIP TF, from which the NLS was obtained. This gene, PITG\_11668, encodes a TF that was previously shown to have strong nuclear localization in \textit{P. infestans} (Gamboa-Melendez et al. 2013). Next, we replaced the \textit{P. sojae} NLS in vectors pYF2.3-PsNLS-Cas9 and pYF2.3-PsNLS-Cas9-GFP with the NLS region of PITG\_11668 (PiNLS) to obtain the vectors pJH2.6-PiNLS-Cas9 and pJH2.6-PiNLS-Cas9-GFP, respectively (\textbf{Figure ~\ref{figS3_Plasmids}}). In addition, PiNLS was cloned N-terminally of GFP in the basic expression vector pGFP-N (Ah-Fong \& Judelson 2011), to obtain pGFP-PiNLS-GFP (\textbf{Figure ~\ref{figS3_Plasmids}}). Next, these constructs were used for transformation of \textit{P. infestans}. We performed cotransformations of pJH2.6-PiNLS-Cas9 along with gRNAs for \textit{Avr1}, \textit{PiTubA2}, or \textit{PiAP5} carried by vectors pYF2.3-gRNA or pJH2.4-gRNA-Hyg. Unfortunately, for none of the combinations we did obtain transformed lines in which we could observe Cas9 activity, nor did we obtain transformed lines exhibiting a fluorescent signal.

\subsubsection{gRNA expression}
In the \textit{P. sojae }CRISPR/Cas9 system, expression of the ribozyme-gRNA construct as well as that of \textit{nptII}, the gene providing geneticin resistance, is driven by p\textit{Ps}RPL41, the promoter of the \textit{P. sojae }ribosomal gene \textit{Ps}RPL41. Thus, geneticin-resistant transformants should in principle express the ribozyme-gRNA construct. However, in the numerous geneticin-resistant transformants that we have obtained, we did not observe CRISPR/Cas9 activity. Consequently, we questioned whether the promoter of \textit{Ps}RPL41 is active in \textit{P. infestans}. In an earlier study performed in \textit{P. infestans}, the expression stability of the \textit{P. infestans }ortholog of p\textit{Ps}RPL41 was found to be inferior to other \textit{P. infestans} and \textit{P. capsici }ribosomal promoters (Poidevin et al. 2015). p\textit{Ps}RPL41 was not evaluated. To assess whether the use of a different promoter driving the expression of the ribozyme-gRNA construct could improve CRISPR/Cas9 activity, we opted to use the \textit{P. capsici} S9 promoter (p\textit{Pc}S9). The expression of this promoter was found to provide high and stable expression in \textit{P. infestans }(Poidevin et al. 2015). To replace p\textit{Ps}RPL41 with p\textit{Pc}S9, the promotor was PCR amplified from pTOR-S9 (Poidevin et al. 2015) and cloned into EcoRI/NheI sites of pYF2.3-gRNA-Ribo. However, an unexpected NheI site at approximately 150 bp upstream of the 3’ end of the promoter sequence, gave rise to a truncated insert, as the same restriction enzyme is used for gRNA insertion. To circumvent this problem p\textit{Pc}S9 and the gRNA can be introduced into the vector backbone by Gibson assembly but due to time constraints we have not been able to obtain a suitable construct with p\textit{Pc}S9 for transformation of\textit{ P. infestans}.

\subsubsection{Delivery of ribonucleoprotein (RNP) complexes}
In several organisms it has been shown that the delivery of Cas9 protein-gRNA ribonucleoproteins (RNPs) complexes is an efficient method for inducing targeted genome editing events, including numerous plant species (Woo et al. 2015), the nematode \textit{Caenorhabditis elegans} (Cho et al. 2013), filamentous fungi and yeast\textit{ }(Pohl et al. 2016; Grahl et al. 2017), protozoa (Soares Medeiros et al. 2017), and also in human cell lines (Ramakrishna et al. 2014). Moreover, it was found to result in genome editing with substantially higher specificity compared to DNA transformation (Zuris et al. 2015). From a technical perspective, the use of RNP complexes may have several advantages over a ‘regular’ CRISPR system where both components are delivered as plasmid DNA and integrated into the genome. Firstly, the genome-editing efficiency of Cas9 using RNP complexes does not rely on the transcription rate by the host, as both components are preassembled \textit{in vitro} prior to delivery into cells. In \textit{Phytophthora }spp., integration of foreign DNA occurs randomly, and it is plausible that integration in a ‘silent’ part of the genome results in reduced transcription rates. The integration can also lead to disruption of genes, when the transgene is inserted in an open reading frame. Secondly, and more importantly, transgene expression of Cas9 can have adverse effects and can even lead to cell death due to toxicity (Jiang et al. 2014; Kim et al. 2014; Peng et al. 2014). The use of RNP complexes of CRISPR-induced mutations might (partially) alleviate this issue, as after some time the RNP complexes are degraded, reducing chances for side effects.

To our knowledge, the use of RNP complexes for targeted genome editing has not been explored in oomycetes. To assess whether RNP complexes can be used for targeted genome editing in \textit{P. infestans}, we used a modified protocol for PEG-mediated protoplast transformation, substituting plasmid DNA with pre-assembled RNP complexes. In parallel, one sample was co-transfected with the \textit{Avr1} HDR construct (\textbf{Figure ~\ref{fig1_targets}a}), which has \textit{HygB} as a selectable marker. After transfection, regenerated protoplasts were analyzed by a T7EI assay and sequencing. These analyses showed all samples to be identical to control transformants, with no significantly overrepresented aberrant background signals in the sequence chromatograms. Moreover, we could not confirm the integration of \textit{HygB }at the target locus, not even in transformants that were hygromycin-B resistant.

The experimental setup of direct delivery of RNP complexes has some limitations, most importantly the lack of selection. Whereas in a regular plasmid-based transformation experiment the introduced plasmid contains a resistance gene to a selection antibiotic, protein transfection does not yield antibiotic resistant colonies and hence also non-transfected protoplasts will regenerate. Consequently, in our experiments transfected protoplasts might have been overshadowed by non-transfected protoplasts. Moreover, despite careful preparation of the starting material, some residual sporangia may have been present in the protoplast suspension and these also readily overgrow regenerating protoplasts.

Another potential limitation is the nuclear localization of the RNP complexes. The Cas9 protein used in this experiment contains two Simian virus 40 (SV40) T antigen NLS tags, one at the N-terminus and the other at the C-terminus. The SV40 NLS is a well-studied monopartite NLS tag consisting of several basic amino acids (Lange et al. 2007), and has been shown to localize fusion gene products to the nucleus of \textit{P. sojae}, albeit with a reduced efficiency compared to other NLS tags (Fang et al. 2017). Hence, it is expected that the RNP complexes used in this study are localized to the nucleus. Still, the use of a Cas9 protein equipped with a \textit{Phytophthora} specific NLS tag might increase the efficiency (Fang et al. 2017). Since such Cas9 proteins are not commercially available they would have to be produced in-house.

\section{Conclusions and future outlook}
Contrary to expectations set by the successful application of the CRISPR/Cas9 system for targeted genome editing in \textit{P. sojae}, \textit{P. capsici }(B.M. Tyler, personal communication), and \textit{P. palmivora }(Gumtow et al. 2017), we have as yet not been able to implement the system in \textit{P. infestans}. The same holds for colleagues elsewhere who are also experiencing difficulties in effectuating the CRISPR/Cas9 system for use in \textit{P. infestans} (personal communication). It is, however, hard to pinpoint the cause for failure of the system. Likely, it is an additive effect of several suboptimal conditions, such as Cas9 or gRNA expression levels, Cas9 localization, or the incubation temperature. Fang and Tyler (2016) who established the system in \mbox{\textit{P. sojae}} also faced numerous challenges and made substantial modifications in the initial CRISPR/Cas9 procedure to get it to work in \mbox{\textit{P. sojae}}. For \textit{P. palmivora} (Gumtow et al. 2017) choose \textit{Agrobacterium-}mediated transformation (AMT)\textit{ }to implement the CRISPR/Cas9 system. Whereas protoplast transformation typically results in multiple integration events of the transgene(s), AMT usually gives rise to only one or two integrations of the transgene in the genome (Vijn \& Govers 2003). A higher transgene copy number easily results in altered expression levels due to homology-dependent gene silencing or overexpression of the transgene.  However, \textit{P. infestans }transformants resulting from AMT rarely show silencing or overexpression of the target gene (P.J.I. van de Vondervoort, unpublished data), a phenomenon that might be due to the low integration rate. Overall, generating transformants via AMT is less laborious and requires less starting materials than PEG mediated protoplast transformations. We are currently exploring the use of AMT for integration of the CRISPR/Cas9 components in \textit{P. infestans}.

A clear difference between \textit{P. infestans} and the three \textit{Phytophthora} spp. in which CRISPR-induced mutations are observed, is their growth temperature. Whereas \textit{P. infestans} is typically incubated at 18°C, the other three species are grown at 25°C. The Cas9 isoform used in the \textit{P. sojae} CRISPR/Cas9 system is a human codon optimized gene from \textit{Streptococcus pyogenes }(SpCas9), an organism which grows at 37°C. Hence, it is anticipated that the activity of SpCas9 is reduced at decreased temperatures. Indeed, \textit{Arabidopsis thaliana} plants exposed to heat stress (37°C for 30 h) showed much higher rates of targeted mutagenesis by CRISPR/Cas9 compared to plants grown continuously at the standard temperature of 22°C (Le Blanc et al. 2017). On the other hand CRISPR/Cas9 has been employed in salmon eggs at temperatures as low as 6°C (Edvardsen et al. 2014), making it unlikely that the incubation temperature is the sole reason for absence of Cas9 activity in \textit{P. infestans}. Still, incubating potential \textit{P. infestans} transformants at elevated temperatures or exposing them to heat stress, might improve the efficiency of the system and is worth to try. This would require some pilot experiments for determining the right temperature and time of exposure to the heat shock. When \textit{P. infestans }is cultivated at 28°C it dies within 24 hours, but possibly a shorter exposure at elevated temperatures is sufficient to obtain higher frequencies of CRISPR-induced mutations. 

Another possible modification to the system which might be considered, is the use of a different nuclease. \textit{S. pyogenes} Cas9 (SpCas9) is a relatively large protein which might hamper nuclear import. For a smaller isoform, such as Cas9 from \textit{Staphylococcus aureus} (SaCas9), with approximately three quarters of the size of SpCas9 (Ran et al. 2015), it might be easier to find its way to the nucleus. In the protozoa \textit{Trypanosoma cruzi}, delivery of RNP complexes of SaCas9 and gRNA resulted in gene edits, while SpCas9 did not (Soares Medeiros et al. 2017). Likewise, RNP complexes with SaCas9 could be more effective in \textit{P. infestans }compared to those with SpCas9. Also for plasmid-based delivery of the CRISPR/Cas9 components, the smaller gene size of \textit{SaCas9} might improve integration. However, SaCas9 recognizes a different PAM than SpCas9 (NNGRRT vs. NGG). Hence, different gRNAs have to be designed when utilizing this nuclease, and there is likely a lower frequency of the PAM in the sequence of the target gene. Another nuclease to consider is Cpf1 (also known as Cas12a). Whereas SpCas9 creates DSB breaks with 'blunt' ends, Cpf1 introduces a staggered DSB break with a 4 or 5 nt 5' overhang (Zetsche et al. 2015). These cohesive ('sticky') ends can increase the frequency of HDR repair. Moreover, on certain positions, Cpf1 is sensitive to single-base mismatches in the protospacer, reducing the frequency of off-target cleavage (Kleinstiver et al. 2016). However, its T-rich PAM (TTTN) may limit the number of target sites in \textit{P. infestans}, which has a GC content of 51\% (Haas et al. 2009).

We trust that, with dedicated effort, developing a CRISPR/Cas system for \textit{P. infestans} is attainable. Future work should focus on systematic analysis of factors limiting the efficiency of the system. When these limitations are identified and overcome, targeted mutagenesis in \textit{P. infestans} might be within reach.

\section{Acknowledgements}
We are grateful to Yufeng 'Francis' Fang and Brett Tyler for generously sharing constructs and experiences. Thomas van Ravensteyn for helpful discussions. Juliette Silven, Paul Schaap, and Natalie Verbeek-de Kruif are acknowledged for their technical assistance. This work was supported by Division for Earth and Live Sciences (ALW) with financial aid from the Netherlands Organization for Scientific Research (NWO) in the framework of the ALW-JSTP programme (project \# 833.13.002; J.H.; F.G).

\section{Materials and methods}

\begin{footnotesize}

\subsection{Strains, culture conditions and transformations}
\textit{P. infestans} strain T30-2 (van der Lee et al. 2001) and all transgenic lines were routinely grown at 18 °C in the dark on rye agar medium supplemented with 2\% sucrose (RSA) (Caten \& Jinks 1968). RSA was supplemented with 20 μg/ml vancomycin, 100 μg/ml ampicillin and 50 μg/ml amphotericin B, and in addition, for transformed lines with 2.5 μg/ml G418. Transient and stable transformants of \textit{P. infestans} were generated using PEG/CaCl2-mediated protoplast transformation and zoospore electroporation. Protoplast transformation was performed according to methods described previously (Ah-Fong et al. 2008), omitting the step of complexing circular plasmid DNA with Lipofectin in protoplast transformation. Zoospore electroporation was performed following protocols available. 

\subsection{gRNA design}
Sequences for target genes were downloaded from FungiDB (Stajich et al. 2012). The full-length gene products were PCR amplified and subsequently validated by sequencing (Eurofins, Ebersberg, Germany). Next, the sequence was screened for CRISPR sites using NGG as PAM on Geneious R9.1.4 (Kearse et al. 2012). The resulting sites were scored for on-target activity according to Doench \textit{et al}. (Doench et al. 2014), and for off-target interactions according to Hsu et al. (2013). The top 10 best scoring CRISPR sites were manually curated using BLAST analysis on the \textit{P. infestans }reference genome and predictions of secondary RNA structure of the corresponding gRNAs using RNAstructure (Reuter \& Mathews 2010).

gRNAs for \textit{Avr1} and \textit{PiTubA1} were ordered as sense and antisense PAGE-purified oligos (Integrated DNA Technologies) and subsequently annealed and ligated into pYF2.3-gRNA-Ribo-EV, as described by Fang and Tyler (2016), using BsaI and NheI-HF (New England Biolabs). For \textit{PiAP5 }and \textit{PpAP5}, 283 bp constructs transcribing the ribozyme-gRNA insert flanked by two 30 bp overlaps, were designed and ordered as gBlocks (Integrated DNA Technologies). Next, the fragments were inserted into pYF2.3-gRNA-Ribo-EV by Gibson assembly using the NEBuilder HiFi kit (New England Biolabs Inc).

\subsection{Constructs}
The hygromycin-B resistance gene was amplified from pGFP-H (Ah-Fong \& Judelson 2011) using primers Hyg\_AflII\_F and Hyg\_ApaI\_R, and the resulting amplicon was cloned into pYF2.3-gRNA-Ribo using AflI and ApaI (New England Biolabs), yielding pJH2.4-Hyg-gRNA. To constitute the 'all-in-one' vector pJH2.5-Cas9-gRNA, the gRNA expressing construct from pYF2.3-gRNA (containing the RPL41 promoter, gRNA insert, and Hsp70 terminator) was inserted into pYF2.2-Cas9 using EcoRI (Promega). A dual-gRNA vector encoding Avr1-targeting gRNA2 and gRNA3 was constructed by inserting the ribozyme-gRNA3 construct into pYF2.3-gRNA2 by Gibson assembly. The amplicon containing the insert was made using primers gRNA3\_Gibson\_F and gRNA3\_Gibson\_R. The NLS region from PITG\_11668 was inserted at the N-terminus of GFP, the PCR amplicon was cloned into the AgeI and NheI sites of pGFP-N (Ah-Fong \& Judelson 2011), yielding pGFP-PiNLS-GFP. To obtain pJH2.6-PiNLS-Cas9 and pJH2.6-PiNLS-Cas9-GFP, the NLS region from PITG\_11668 was inserted into the SacII and SpeI sites of pYF2.2-PsNLS-Cas9 and pYF2.2-PsNLS-Cas9-GFP. ΔAvr1 was constructed by overlap extension PCR using primers Avr1\_del\_F and Avr1\_del\_R. All primers used in this study are listed in Table ~\ref{TabS1_primers}. 

\subsection{Cas9 \textit{in vitro} activity assay}
To obtain template DNA for \textit{in vitro} transcription (IVT), a PCR was performed with a T7 promoter-fused forward primer (marked with extension \_pT7\_F) and the primer sgRNA\_Col\_R, using plasmid DNA containing the respective gRNAs as template. Next, gRNA was transcribed with T7 RNA polymerase using MEGAshortscript T7 kit (Thermo Fischer Scientific). IVT was allowed to proceed for 4 h, after which RNA was purified by phenol/chloroform and ethanol precipitation, and analyzed on agarose gel. Target DNA was amplified using the respective primers for full-length PCR products of \textit{Avr1}, \textit{PiTubA2}, and \textit{PiAP5}. SpCas9 nuclease was purchased (New England Biolabs). The assay was performed according to manufacturer’s instructions.

\subsection{Molecular analysis of transformants}
Genomic DNA (gDNA) was extracted from pooled or individual \textit{P. infestans} transformants according to methods described previously (Fang \& Tyler 2016), with modifications. Pooled transformants, 24-48 h after transformation, were pelleted by centrifugation, resuspended in 500 mL of gDNA extraction buffer (200 mM Tris, pH 8.0, 200 mM NaCl, 25 mM EDTA, pH 8.0, 2\% SDS, plus 0.1 mg/mL RNase A added prior to use) and sheared by vigorous pipetting. For individual transformants, approximately 250 µl of \textit{P. infestans }mycelium was frozen in liquid nitrogen, freeze-dried, and ground to a powder using a Retsch Mixer Mill MM 400 and metal beads (ø 3 mm) for 30 seconds at 30 Hz. Subsequently, the powder was resuspended in 500 μl of gDNA extraction buffer. DNA was recovered by phenol/chloroform extraction and isopropanol precipitation. RNA was isolated using home-made TRIzol (Verdonk 2014), and cDNA was synthesized using M-MLV Reverse Transcriptase (Promega) according to manufacturer’s instructions. RT-PCR was performed using primers Cas9\_RT\_F and Cas9\_RT\_R.

All PCR amplifications were conducted using Q5 high-fidelity DNA polymerase (New England Biolabs). Nested PCR was performed using diluted (1000x) PCR products as a DNA template. PCR amplicons were purified using the NucleoSpin Gel and PCR Clean-up kit (Macherey-Nagel). Sequence analysis was performed at Eurofins. 

\subsection{T7EI assay}
The efficiency of the CRISPR/Cas9 system was tested with T7 endonuclease I (T7EI) (New England Biolabs), using gDNA isolated from pooled transformants as template. First, the target gene was amplified from gDNA using primers amplifying the full-length sequence of the respective target genes (Table S1). Next, purified amplicons were annealed in a thermocycler using the following conditions: 95°C for 5 min, ramp down to 85°C at -2 °C/s, ramp down to 20°C at -0.1 °C/s, hold at room temperature. The T7EI cleavage assay was performed according to manufacturer’s instructions and analyzed by gel electrophoresis. 

\subsection{RNP assembly and transfection}
Ribonucleoprotein complexes (RNPs) of Cas9 (EnGen Cas9 NLS, New England Biolabs) and gRNAs were prepared immediately before transfections. After IVT, gRNAs were refolded by heating at 90°C for 5 min and cooling to room temperature over the course of 12 h. To preassemble RNPs, equimolar amounts of Cas9 (120 pmol) and gRNA (120 pmol) were incubated at 25°C for 10 min. Next, RNPs (10 µl) were incubated with an equal volume of lipofectin for 5 min at room temperature and added to a 100 µl protoplast suspension (2 x 106/ml). After 5 minutes, 120 µl freshly prepared PEG solution (50\% PEG-3350, 10 mM CaCl2, 10 mM Tris-HCl pH7.5) was added slowly. After incubation for 5 min at room temperature the volume was adjusted to 10 ml with regeneration medium (RSM + 1 M mannitol, without antibiotics) and protoplasts were regenerated for 48 h at 22°C. Next, regenerated protoplasts were collected by centrifugation, from which a sample was taken for gDNA extraction, while the remainder was plated on RSA plates. Protoplasts co-transfected with a HDR construct were plated on RSA plates supplemented with 25 mg/ml hygromycin-B. Colonies appeared on non-selective plates after two days, or after four days on selective plates. Individual colonies were transferred to new plates as soon as they appeared and cultured for further analysis.

\end{footnotesize}

\section{References}
\setlength\parindent{-0.5 em}
\setlength{\parskip}{0 em}
\setlength{\leftskip}{0.5 em}

\begin{sloppy}
\begin{footnotesize}
Ah-Fong, A. M., C. A. Bormann-Chung and H. S. Judelson (2008). Optimization of transgene-mediated silencing in \textit{Phytophthora infestans} and its association with small-interfering RNAs, \textit{Fungal Genet Biol} \textbf{45}: 1197-1205. doi: 10.1016/j.fgb.2008.05.009

Ah-Fong, A. M. and H. S. Judelson (2011). Vectors for fluorescent protein tagging in \textit{Phytophthora}: tools for functional genomics and cell biology, \textit{Fungal Biol} \textbf{115}: 882-890. doi: 10.1016/j.funbio.2011.07.001

Brinkman, E. K., T. Chen, M. Amendola and B. van Steensel (2014). Easy quantitative assessment of genome editing by sequence trace decomposition, \textit{Nucleic Acids Res} \textbf{42}: e168. doi: 10.1093/nar/gku936

Caten, C. E. and J. L. Jinks (1968). Spontaneous Variability of Single Isolates of \textit{Phytophthora infestans}. Cultural Variation, \textit{Can J Botany} \textbf{46}: 329-348. doi: 10.1139/b68-055

Chen, F., S. M. Pruett-Miller, Y. Huang, M. Gjoka, K. Duda, J. Taunton, . . . G. D. Davis (2011). High-frequency genome editing using ssDNA oligonucleotides with zinc-finger nucleases, \textit{Nat Methods} \textbf{8}: 753-755. doi: 10.1038/nmeth.1653

Cho, S. W., J. Lee, D. Carroll, J. S. Kim and J. Lee (2013). Heritable gene knockout in \textit{Caenorhabditis elegans} by direct injection of Cas9-sgRNA ribonucleoproteins, \textit{Genetics} \textbf{195}: 1177-1180. doi: 10.1534/genetics.113.155853

Doench, J. G., E. Hartenian, D. B. Graham, Z. Tothova, M. Hegde, I. Smith, . . . D. E. Root (2014). Rational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation, \textit{Nat Biotechnol} \textbf{32}: 1262-1267. doi: 10.1038/nbt.3026

Du, Y., R. Weide, Z. Zhao, P. Msimuko, F. Govers and K. Bouwmeester (2018). RXLR effector diversity in Phytophthora infestans isolates determines recognition by potato resistance proteins; the case study AVR1 and R1, \textit{Studies in Mycology}. doi: 10.1016/j.simyco.2018.01.003

Edvardsen, R. B., S. Leininger, L. Kleppe, K. O. Skaftnesmo and A. Wargelius (2014). Targeted mutagenesis in Atlantic salmon (\textit{Salmo salar L.}) using the CRISPR/Cas9 system induces complete knockout individuals in the F0 generation, \textit{PLoS One} \textbf{9}: e108622. doi: 10.1371/journal.pone.0108622

Fang, Y., H. S. Jang, G. W. Watson, D. P. Wellappili and B. M. Tyler (2017). Distinctive Nuclear Localization Signals in the Oomycete \textit{Phytophthora sojae}, \textit{Front Microbiol} \textbf{8}: 10. doi: 10.3389/fmicb.2017.00010

Fang, Y. and B. M. Tyler (2016). Efficient disruption and replacement of an effector gene in the oomycete \textit{Phytophthora sojae} using CRISPR/Cas9, \textit{Mol Plant Pathol} \textbf{17}: 127-139. doi: 10.1111/mpp.12318

Fry, W. (2008). \textit{Phytophthora infestans}: the plant (and R gene) destroyer, \textit{Mol Plant Pathol} \textbf{9}: 385-402. doi: 10.1111/j.1364-3703.2007.00465.x

Gamboa-Melendez, H., A. I. Huerta and H. S. Judelson (2013). bZIP transcription factors in the oomycete \textit{Phytophthora infestans} with novel DNA-binding domains are involved in defense against oxidative stress, \textit{Eukaryot Cell} \textbf{12}: 1403-1412. doi: 10.1128/EC.00141-13

Govers, F. and M. Gijzen (2006). \textit{Phytophthora }genomics: the plant destroyers' genome decoded, \textit{Mol Plant Microbe Interact} \textbf{19}: 1295-1301. doi: 10.1094/MPMI-19-1295

Grahl, N., E. G. Demers, A. W. Crocker and D. A. Hogan (2017). Use of RNA-Protein Complexes for Genome Editing in Non-albicans \textit{Candida} Species, \textit{mSphere} \textbf{2}. doi: 10.1128/mSphere.00218-17

Gumtow, R., D. Wu, J. Uchida and M. Tian (2017). A \textit{Phytophthora palmivora} extracellular cystatin-like protease inhibitor targets papain to contribute to virulence on papaya, \textit{Mol Plant Microbe Interact}. doi: 10.1094/MPMI-06-17-0131-FI

Haas, B. J., S. Kamoun, M. C. Zody, R. H. Jiang, R. E. Handsaker, L. M. Cano, . . . C. Nusbaum (2009). Genome sequence and analysis of the Irish potato famine pathogen \textit{Phytophthora infestans}, \textit{Nature} \textbf{461}: 393-398. doi: 10.1038/nature08358

Hsu, P. D., D. A. Scott, J. A. Weinstein, F. A. Ran, S. Konermann, V. Agarwala, . . . F. Zhang (2013). DNA targeting specificity of RNA-guided Cas9 nucleases, \textit{Nat Biotechnol} \textbf{31}: 827-832. doi: 10.1038/nbt.2647

Jiang, W., A. J. Brueggeman, K. M. Horken, T. M. Plucinak and D. P. Weeks (2014). Successful transient expression of Cas9 and single guide RNA genes in \textit{Chlamydomonas reinhardtii}, \textit{Eukaryot Cell} \textbf{13}: 1465-1469. doi: 10.1128/EC.00213-14

Judelson, H. and B. M. Tyler. (2001). "Transformation of \textit{Phytohpthora infestans }by electroporation." from https://oomyceteworld.net/Phytophthora electroporatio.pdf.

Judelson, H. S. (1997). The genetics and biology of \textit{Phytophthora infestans}: modern approaches to a historical challenge, \textit{Fungal Genet Biol} \textbf{22}: 65-76. doi: 10.1006/fgbi.1997.1006

Kamoun, S., O. Furzer, J. D. Jones, H. S. Judelson, G. S. Ali, R. J. Dalio, . . . F. Govers (2015). The Top 10 oomycete pathogens in molecular plant pathology, \textit{Mol Plant Pathol} \textbf{16}: 413-434. doi: 10.1111/mpp.12190

Kay, J., H. J. G. Meijer, A. ten Have and J. A. van Kan (2011). The aspartic proteinase family of three \textit{Phytophthora }species, \textit{BMC Genomics} \textbf{12}: 254. doi: 10.1186/1471-2164-12-254

Kearse, M., R. Moir, A. Wilson, S. Stones-Havas, M. Cheung, S. Sturrock, . . . A. Drummond (2012). Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data, \textit{Bioinformatics} \textbf{28}: 1647-1649. doi: 10.1093/bioinformatics/bts199

Kim, S., D. Kim, S. W. Cho, J. Kim and J. S. Kim (2014). Highly efficient RNA-guided genome editing in human cells via delivery of purified Cas9 ribonucleoproteins, \textit{Genome Res} \textbf{24}: 1012-1019. doi: 10.1101/gr.171322.113

Kleinstiver, B. P., S. Q. Tsai, M. S. Prew, N. T. Nguyen, M. M. Welch, J. M. Lopez, . . . J. K. Joung (2016). Genome-wide specificities of CRISPR-Cas Cpf1 nucleases in human cells, \textit{Nat Biotechnol} \textbf{34}: 869-874. doi: 10.1038/nbt.3620

Lange, A., R. E. Mills, C. J. Lange, M. Stewart, S. E. Devine and A. H. Corbett (2007). Classical nuclear localization signals: definition, function, and interaction with importin alpha, \textit{J Biol Chem} \textbf{282}: 5101-5105. doi: 10.1074/jbc.R600026200

Le Blanc, C., F. Zhang, J. Mendez, Y. Lozano, K. Chatpar, V. Irish and Y. Jacob (2017). Increased efficiency of targeted mutagenesis by CRISPR/Cas9 in plants using heat stress, \textit{Plant J}. doi: 10.1111/tpj.13782

Ma, Z., L. Zhu, T. Song, Y. Wang, Q. Zhang, Y. Xia, . . . Y. Wang (2017). A paralogous decoy protects \textit{Phytophthora sojae} apoplastic effector PsXEG1 from a host inhibitor, \textit{Science} \textbf{355}: 710-714. doi: 10.1126/science.aai7919

Peng, D., S. P. Kurup, P. Y. Yao, T. A. Minning and R. L. Tarleton (2014). CRISPR-Cas9-mediated single-gene and gene family disruption in \textit{Trypanosoma cruzi}, \textit{mBio} \textbf{6}: e02097-02014. doi: 10.1128/mBio.02097-14

Pohl, C., J. A. Kiel, A. J. Driessen, R. A. Bovenberg and Y. Nygard (2016). CRISPR/Cas9 Based Genome Editing of \textit{Penicillium chrysogenum}, \textit{ACS Synth Biol} \textbf{5}: 754-764. doi: 10.1021/acssynbio.6b00082

Poidevin, L., K. Andreeva, C. Khachatoorian and H. S. Judelson (2015). Comparisons of Ribosomal Protein Gene Promoters Indicate Superiority of Heterologous Regulatory Sequences for Expressing Transgenes in \textit{Phytophthora infestans}, \textit{PLoS One} \textbf{10}: e0145612. doi: 10.1371/journal.pone.0145612

Ramakrishna, S., A. B. Kwaku Dad, J. Beloor, R. Gopalappa, S. K. Lee and H. Kim (2014). Gene disruption by cell-penetrating peptide-mediated delivery of Cas9 protein and guide RNA, \textit{Genome Res} \textbf{24}: 1020-1027. doi: 10.1101/gr.171264.113

Ran, F. A., L. Cong, W. X. Yan, D. A. Scott, J. S. Gootenberg, A. J. Kriz, . . . F. Zhang (2015). In vivo genome editing using Staphylococcus aureus Cas9, \textit{Nature} \textbf{520}: 186-191. doi: 10.1038/nature14299

Ran, F. A., P. D. Hsu, J. Wright, V. Agarwala, D. A. Scott and F. Zhang (2013). Genome engineering using the CRISPR-Cas9 system, \textit{Nat Protoc} \textbf{8}: 2281-2308. doi: 10.1038/nprot.2013.143

Reuter, J. S. and D. H. Mathews (2010). RNAstructure: software for RNA secondary structure prediction and analysis, \textit{BMC Bioinformatics} \textbf{11}: 129. doi: \mbox{10.1186/1471-2105-11-129}

Soares Medeiros, L. C., L. South, D. Peng, J. M. Bustamante, W. Wang, M. Bunkofske, . . . R. L. Tarleton (2017). Rapid, Selection-Free, High-Efficiency Genome Editing in Protozoan Parasites Using CRISPR-Cas9 Ribonucleoproteins, \textit{mBio} \textbf{8}. doi: 10.1128/mBio.01788-17

Stajich, J. E., T. Harris, B. P. Brunk, J. Brestelli, S. Fischer, O. S. Harb, . . . D. S. Roos (2012). FungiDB: an integrated functional genomics database for fungi, \textit{Nucleic Acids Res} \textbf{40}: D675-681. doi: 10.1093/nar/gkr918

Van den Hoogen, D. J., H. J. G. Meijer, M. F. Seidl and F. Govers (2018). The ancient link between G-protein coupled receptors and C-terminal phospholipid kinase domains, \textit{mBio} \textbf{9}. doi: 10.1128/mBio.02119-17

van der Lee, T., A. Robold, A. Testa, J. W. van 't Klooster and F. Govers (2001). Mapping of avirulence genes in \textit{Phytophthora infestans} with amplified fragment length polymorphism markers selected by bulked segregant analysis, \textit{Genetics} \textbf{157}: 949-956. 

van West, P., S. Kamoun, J. W. van 't Klooster and F. Govers (1999). Internuclear gene silencing in Phytophthora infestans, \textit{Mol Cell} \textbf{3}: 339-348.

Verdonk, J. (2014). RNA isolation with homemade TRIzol reagent. from https://julianverdonk.wordpress.com/2014/01/14/rna-isolation-with-homemade-trizol-reagent/.

Vijn, I. and F. Govers (2003). Agrobacterium tumefaciens mediated transformation of the oomycete plant pathogen Phytophthora infestans, \textit{Mol Plant Pathol} \textbf{4}: 459-467. doi: 10.1046/j.1364-3703.2003.00191.x

Woo, J. W., J. Kim, S. I. Kwon, C. Corvalan, S. W. Cho, H. Kim, . . . J. S. Kim (2015). DNA-free genome editing in plants with preassembled CRISPR-Cas9 ribonucleoproteins, \textit{Nat Biotechnol} \textbf{33}: 1162-1164. doi: 10.1038/nbt.3389

\end{footnotesize}
\end{sloppy}
\end{multicols}


\clearpage

\section{Supplementary files}
\makeatletter 
\setcounter{figure}{0}
\renewcommand{\thetable}{S\arabic{table}}   
\renewcommand{\thefigure}{S\arabic{figure}}
\renewcommand{\figurename}{Supplemental Figure}
\renewcommand{\tablename}{Supplemental Table}

\begin{figure*}[h]
\centering
\includegraphics[width=0.7\textwidth]{Figures/FigS1}
\caption{\textbf{Detection limit assays.} \textbf{a)} PCR amplicons used for the detection limit assays. ΔAvr1 contains a 29 bp deletion. \textbf{b)} T7EI assay. Varying amounts of Avr1 and ΔAvr1 PCR amplicons were annealed and digested by T7EI. \textbf{c)} Example output from TIDE, confirming the presence of the 29 bp deletion in ΔAvr1 (left red bar). Here, a sequence chromatogram obtained from sequencing a molar ratio of 998:2 Avr1:ΔAvr1 was compared to a sequence chromatogram of Avr1. The predicted frequency of chromatograms with a deletion (1.5\%) deviates from the actual molar ratio of Avr1:ΔAvr1 (0.2\%).}
\label{figS1_detect}
\end{figure*}

\begin{figure*}[h]
\centering
\includegraphics[width=\textwidth]{Figures/FigS2}
\caption{\textbf{Expression analysis of Cas9 in 11 selected transformed lines.} cDNA and genomic DNA (gDNA) of\textit{ P. infestans }strain T30-2 were used as negative control and gDNA from a \textit{P. infestans} Cas9 transformant as positive control.}
\label{figS2_RT}
\end{figure*}

\begin{figure*}[h]
\centering
\includegraphics[width=\textwidth]{Figures/FigS3}
\caption{\textbf{Plasmids used in this study.} Plasmids starting with `pYF' are obtained from Francis Fang (Fang \& Tyler 2017), plamids starting with `pJH' are modified versions as described in this report.}
\label{figS3_Plasmids}
\end{figure*}

\begin{table*}[h]
\rowcolors{0}{}{lightgray}
\centering
\caption{\textbf{Primers used in this study}.}
\label{TabS1_primers}
\resizebox{\textwidth}{!}{%
\begin{tabular}{llll}
\hline
\textbf{Name}                     & \textbf{Sequence (5' - 3')}                           & \textbf{Purpose}                                   &  \\ \hline
Avr1\_F                  & ATGCACCGCGTATTGCTGC                          & \textit{P. infestans} T30-2 Avr1          &  \\
Avr1\_F                  & TTAAAATGGTACCACAACATGTCCACC                  & \textit{P. infestans} T30-2 Avr1          &  \\
Avr1\_Nested F           & GACGTTTGCCCTGTTGTGTA                         & \textit{P. infestans} T30-2 Avr1 - nested &  \\
Avr1\_Nested R           & ACAACATGTCCACCAAGCATG                        & \textit{P. infestans} T30-2 Avr1 - nested &  \\
pRPL41\_seq\_F           & CAAGCCTCACTTTCTGCTGACTG                      & Sequence analysis of gRNA constructs      &  \\
sgRNA\_Col\_R            & AAAAGCACCGACTCGGTGC                          & Sequence analysis of gRNA constructs      &  \\
Cas9\_RT\_F              & CCGAAGAGGTCGTGAAGAAG                         & Expression analysis of Cas9               &  \\
Cas9\_RT\_R              & GCCTTATCCAGTTCGCTCAG                         & Expression analysis of Cas9               &  \\
pHAM34\_Pseq\_F          & TCGCCCGACTCGCCCAC                            & Sequence analysis of Cas9                 &  \\
Cas9\_Seq\_757\_F        & CTGTTCGGAAACCTGATTGCCC                       & Sequence analysis of Cas9                 &  \\
Cas9\_Seq\_1439\_F       & GGATGACCAGAAAGAGCGAGG                        & Sequence analysis of Cas9                 &  \\
Cas9\_Seq\_2163\_F       & AGAGGACATCCAGAAAGCCCAGG                      & Sequence analysis of Cas9                 &  \\
Cas9\_Seq\_2865\_F       & GAACACTAAGTACGACGAGAATGAC                    & Sequence analysis of Cas9                 &  \\
Cas9\_Seq\_3600\_F       & CAAGGGCTACAAAGAAGTGAAAAAG                    & Sequence analysis of Cas9                 &  \\
eGFP\_F                  & ATGGGCAAGGGCGAGGAA                           & Detection of eGFP                         &  \\
eGFP\_R                  & TCACTTGTAGAGTTCATCCATGCCA                    & Detection of eGFP                         &  \\
Hyg\_ApaI\_R             & GGGCCCCTATTCCTTTGCCCTC                       & Cloning Hygromycin into pYF2.3            &  \\
Hyg-AflII\_F             & CTTAAGATGAAAAAGCCTGAACTCAC                   & Cloning Hygromycin into pYF2.3            &  \\
M13F                     & CGTTGTAAAACGACGGCCAG                         & General primers                           &  \\
M13R                     & TGCCAGGAAACAGCTATGACC                        & General primers                           &  \\
PiNLS\_SacII\_F          & ACACCCGCGGATGCACAAGCGCAAG                    & Cloning PiNLS in pYF2.2                   &  \\
PiNLS\_SpeI\_R           & ACACACTAGTCTCGCCCATTGCCGCGTC                 & Cloning PiNLS in pYF2.2                   &  \\
PiNLS\_AgeI\_F           & ACACACCGGTATGCACAAGCGCAAG                    & Cloning PiNLS in pGFPN                    &  \\
PiNLS\_NheI\_R           & ACACGCTAGCCTCGCCCATTGCCGCGTC                 & Cloning PiNLS in pGFPN                    &  \\
PiAP5\_EcoRI\_F          & ACACGAATTCATGCGTCTCGGTCTGCTC                 & PiAp5 full-length                         &  \\
PiAP5\_NotI\_R           & ACACGCGGCCGCATTTGGTCCCATGAGACGCG             & PiAp5 full-length                         &  \\
PcS9\_F\_EcoRI           & cacaGAATTCCGTCAATACGGCTGTAAACCAC             & \textit{P. capsici S9} promoter          &  \\
PcS9\_F\_NheI            & cacaGCTAGCTTTGGCGACTTCTTTTGTTCAGG            & \textit{P. capsici S9} promoter           &  \\
Avr1\_gRNA\_1\_pT7\_F    & GAAATTAATACGACTCACTATAGGTGGCCAAAGCAATGATATTG & In vitro transcription gRNA               &  \\
Avr1\_gRNA\_2\_pT7\_F    & GAAATTAATACGACTCACTATAGGTCGTTTATCGAGTCCTTCGT & In vitro transcription gRNA               &  \\
Avr1\_gRNA\_3\_pT7\_F    & GAAATTAATACGACTCACTATAGGGAATCCAAGACTCGATTTTT & In vitro transcription gRNA               &  \\
PiTubA2\_gRNA\_8\_pT7\_F & GAAATTAATACGACTCACTATAGGACGCATGTTGCTTTAAGCTT & In vitro transcription gRNA               &  \\
PiAP5\_gRNA\_183\_pT7\_F & GAAATTAATACGACTCACTATAGGGATAAGACGGTGAACAGCAA & In vitro transcription gRNA               &  \\
gRNA3\_Gibson\_F         & TCGGCATGGCGAATGGGACAGGATTCCTGATGAGTCCGTGA    & gRNA2-3 plasmid assembly                   &  \\
gRNA3\_Gibson\_R         & GCTAAGTATTCTAGTCGACAGTCCCATTCGCCATGCCGAA     & gRNA2-3 plasmid assembly                           &  \\ \hline
\end{tabular}%
}
\end{table*}

\end{document}